pkg/cmd/config.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"github.com/puppetlabs/relay/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+func newConfigCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config",
+		Short: "Manage Relay CLI configuration",
+		Args:  cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(newConfigAuthCommand())
+
+	return cmd
+}
+
+func newConfigAuthCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "auth",
+		Short: "Manage Relay CLI authentication configuration",
+		Args:  cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(newConfigAuthClearCommand())
+
+	return cmd
+}
+
+func newConfigAuthClearCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "clear",
+		Short: "Clear the stored authentication for the current context",
+		Args:  cobra.ExactArgs(0),
+		RunE:  doConfigAuthClear,
+	}
+
+	return cmd
+}
+
+func doConfigAuthClear(cmd *cobra.Command, args []string) error {
```

Comment:
It might be nice to pick which token type you want to clear, what do you think?

Reply:
Ahh, yes, definitely. I'll add that.

Reply:
Eventually this will be more robust, and include an easy means to clear out all auth data as well, but that will require future changes to always maintain knowledge of all possible contexts instead of only loading the necessary context on the fly. Which is going to be helpful for a variety of reasons (i.e. listing known contexts), but those changes can/should wait for additional updates, as this is already starting to push the boundaries of reasonable size.